src/emulator/module/include/module/bridge_args.h

```diff
@@ -96,6 +96,37 @@ struct ArgLayout {
     }
 };
 
+template <typename R0, typename StackHead, typename... StackTail>
+struct ArgLayout<R0, long long, StackHead, StackTail...> {
+    using StackLayout2 = StackLayout<StackType<StackHead>, StackType<StackTail>...>;
+    
+    template <typename T, size_t index>
+    static typename std::enable_if < index<2, T>::type read(CPUState &cpu, const MemState &mem) {
+        if(sizeof(T)==sizeof(uint64_t)){
+            uint64_t value = 0;
+            value |= read_reg(cpu,index*2);
```

Comment:
This looks quite fragile but we discussed in Discord that it was a temporary workaround until we make bridging arguments more understandable.